task_templates/pipelines/python3_sklearn_regression/custom.py

```diff
@@ -15,134 +17,91 @@ def fit(
-#     `data` will be the transformed data.
-#     model : is the deserialized model loaded by DRUM or by `load_model`, if supplied
-#     kwargs : additional keyword arguments to the method
-#     In case of classification model class labels will be provided as the following arguments:
-#     - `positive_class_label` is the positive class label for a binary classification model
-#     - `negative_class_label` is the negative class label for a binary classification model
-#
-#     Returns
-#     -------
-#     This method should return predictions as a dataframe with the following format:
-#       Binary Classification: must have columns for each class label with floating- point class
-#         probabilities as values. Each row should sum to 1.0
-#       Regression: must have a single column called `Predictions` with numerical values
-#
-#     """
-
-# def post_process(predictions: pd.DataFrame, model: Any) -> pd.DataFrame:
-#     """
-#     This method is only needed if your model's output does not match the above expectations
-#
-#     Parameters
-#     ----------
-#     predictions : is the dataframe of predictions produced by DRUM or by
-#       the `score` hook, if supplied
-#     model : is the deserialized model loaded by DRUM or by `load_model`, if supplied
-#
-#     Returns
-#     -------
-#     This method should return predictions as a dataframe with the following format:
-#       Binary Classification: must have columns for each class label with floating- point class
-#         probabilities as values. Each row
-#     should sum to 1.0
-#       Regression: must have a single column called `Predictions` with numerical values
-#
-#     """
+    This hook defines the output of a custom estimator and returns predictions on input data.
+    It should be skipped if a task is a transform.
+
+    Note: While best practice is to include the score hook, if the score hook is not present DataRobot will
+    add a score hook and call the default predict method for the library
+    See https://github.com/datarobot/datarobot-user-models#built-in-model-support for details
+
+    Parameters
+    ----------
+    data: pd.DataFrame
+        Is the dataframe to make predictions against. If the `transform` hook is utilized,
+        `data` will be the transformed data
+    model: Any
+        Trained object, extracted by DataRobot from the artifact created in fit().
+        In this example, contains trained sklearn pipeline extracted from artifact.pkl.
+    kwargs:
+        Additional keyword arguments to the method
+
+    Returns
+    -------
+    This method should return predictions as a dataframe with the following format:
+      Binary Classification: must have columns for each class label with floating- point class
+        probabilities as values. Each row should sum to 1.0. The original class names defined in the project
+        must be used as column names.
+      Regression: must have a single column called `Predictions` with numerical values
```

Comment:
Should only have regression here since the pipeline only supports that

Reply:
agreed, done